WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignContentCollectionViewController.swift

```diff
@@ -3,18 +3,23 @@ import UIKit
 import WordPressKit
 
 class SiteDesignContentCollectionViewController: CollapsableHeaderViewController {
-    typealias TemplateGroup = SiteDesignRequest.TemplateGroup
     typealias PreviewDevice = PreviewDeviceSelectionViewController.PreviewDevice
 
+    private let creator: SiteCreator
     private let createsSite: Bool
-    private let templateGroups: [TemplateGroup] = [.stable, .singlePage]
     private let tableView: UITableView
     private let completion: SiteDesignStep.SiteDesignSelection
-    private let restAPI = WordPressComRestApi.anonymousApi(
-        userAgent: WPUserAgent.wordPress(),
-        localeKey: WordPressComRestApi.LocaleKeyV2
-    )
-    private var sections: [SiteDesignSection] = []
+
+    private var sections: [SiteDesignSection] = [] {
+        didSet {
+            if oldValue.count == 0 {
```

Comment:
nit: maybe we can use `isEmpty` instead of explicitly refer to a count value?

Reply:
Sure - this was transplanted from the [previous code](https://github.com/wordpress-mobile/WordPress-iOS/pull/18657/files#diff-2df095be93725951ad9d6172ba658a75d44441f11e491074eeca03046bb38ba8L32), so I'll be happy to make the change.

Reply:
Updated in d08b496fda5ebcd0ddffa0d60b2bfac70b2cc945.